packages/zendesk-adapter/src/filters/article.ts

```diff
@@ -14,18 +14,54 @@
 * limitations under the License.
 */
 import _ from 'lodash'
+import Joi from 'joi'
 import {
-  Change, getChangeData, InstanceElement, isRemovalChange,
+  Change, getChangeData, InstanceElement, isAdditionChange, isRemovalChange, ReferenceExpression,
 } from '@salto-io/adapter-api'
+import { createSchemeGuard } from '@salto-io/adapter-utils'
 import { FilterCreator } from '../filter'
 import { deployChange, deployChanges } from '../deployment'
 import { ARTICLE_TYPE_NAME } from '../constants'
 import { addRemovalChangesId } from './help_center_section_and_category'
 
+const TRANSLATION_SCHEMA = Joi.object({
+  locale: Joi.object().required(),
+  body: Joi.string(),
+  title: Joi.string().required(),
+}).unknown(true).required()
+
+export type TranslationType = {
+  title: string
+  body?: string
+  locale: ReferenceExpression
+}
+
+export const isTranslation = createSchemeGuard<TranslationType>(
+  TRANSLATION_SCHEMA, 'Received an invalid value for translation'
+)
+
+const addTranslationValues = (change: Change<InstanceElement>): void => {
+  const currentLocale = getChangeData(change).value.source_locale
+  const translation = getChangeData(change).value.translations
+    .filter(isTranslation) // the translation is not a reference it is already the value
+    .find((tran: TranslationType) => tran.locale.value.value.id === currentLocale)
+  if (translation !== undefined) {
+    getChangeData(change).value.name = translation.title
+    getChangeData(change).value.description = translation.body ?? ''
```

Comment:
```suggestion
    getChangeData(change).value.title = translation.title
    getChangeData(change).value.body = translation.body ?? ''
```
(please confirm we don't need another mapping)